docs/src/lib/functions.md

```diff
@@ -33,6 +33,11 @@ rename
 rename!
 ```
 
+## Working with views
```

Comment:
Maybe put this in "Summary information" instead?

Reply:
ok - moved.